__tests__/server/middleware/sendHtml.spec.js

```diff
@@ -402,6 +411,41 @@ describe('sendHtml', () => {
       expect(/<script.*nonce="54321"/.test(res.send.mock.calls[0][0])).toBe(true);
     });
 
+    describe('PWA config rendering', () => {
+      it('includes __pwa_metadata__ with disabled values', () => {
+        sendHtml(req, res);
+        expect(res.send).toHaveBeenCalledTimes(1);
+        expect(/window\.__pwa_metadata__ = {"enabled":false,"scope":null,"scriptUrl":null};/.test(res.send.mock.calls[0][0])).toBe(true);
+        expect(/<link rel="manifest" href="\/manifest\.webmanifest">/.test(res.send.mock.calls[0][0])).not.toBe(true);
```

Comment:
Shouldnt the `href` be `/_/pwa/manifest.webmanifest`?

Reply:
I believe here I mocked the value for simplicity